updates/testing.json

```diff
@@ -21,18 +21,18 @@
       }
     },
     {
-      "version": "31.20191211.1",
+      "version": "31.20191217.2.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "31.20191217.2.0",
+      "version": "31.20200108.2.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1576765800,
```

Comment:
i'm hesitant on this. I hate to say it but should we wait til monday?

Reply:
Yeah, I think that makes sense.

Reply:
ok fixed

Reply:
Monday 9AM. :heavy_check_mark:

Reply:
EST, yes :)